app.py

```diff
@@ -45,10 +58,7 @@ def is_sig_valid(sig, msg, pub_key):
 
 
 def get_db_collection(name):
```

Comment:
I don't think this function is needed now that we have the global connection object